packages/genesys-spark-components/src/components/beta/gux-popover-beta/gux-popover.tsx

```diff
@@ -0,0 +1,321 @@
+            padding: 16
+          })
+        ]
+      }).then(({ x, y, middlewareData, placement }) => {
+        Object.assign(this.popupElement.style, {
+          left: `${x}px`,
+          top: `${y}px`
+        });
+
+        const side = placement.split('-')[0];
+
+        const staticSide = {
+          top: 'bottom',
+          right: 'left',
+          bottom: 'top',
+          left: 'right'
+        }[side];
+
+        const arrowRotation = {
+          top: 0,
+          right: 90,
+          bottom: 180,
+          left: -90
+        }[side];
+
+        // This is 13 because this makes the arrow look aligned when horizontal
+        // or 15 if vertical due to extra padding needed to show the shadow.
+        const arrowLen = side === 'left' || side === 'right' ? 15 : 13;
+
+        if (middlewareData.arrow) {
+          let x = middlewareData.arrow.x;
+          const y = middlewareData.arrow.y;
+
+          if (side === 'left' || side === 'right') {
+            x = x + 4;
+          }
+
+          this.popupElement.setAttribute('data-placement', placement);
+
+          Object.assign(this.arrowElement.style, {
+            left: x != null ? `${x}px` : '',
+            top: y != null ? `${y}px` : '',
+            right: '',
+            bottom: '',
+            [staticSide]: `${-arrowLen}px`,
+            transform: `rotate(${arrowRotation}deg)`
+          });
+        }
+      });
+    }
+  }
+
+  private dismiss(): void {
+    const dismissEvent = this.guxdismiss.emit();
+    if (!dismissEvent.defaultPrevented) {
+      this.isOpen = false;
+      this.popupElement.hidePopover();
+    }
+  }
+
+  private logForAttributeError(): void {
+    if (this.root.isConnected) {
+      console.error(
+        `gux-popover: invalid element supplied to 'for': "${this.for}"`
+      );
+    }
+  }
+
+  connectedCallback(): void {
+    this.updateForElement();
+    trackComponent(this.root, { variant: this.position });
+  }
+
+  componentDidLoad(): void {
+    if (!this.forElement) {
+      return;
+    }
+
+    this.popupElement.popover = 'manual';
+    this.forElement.popoverTargetElement = this.popupElement;
+    this.forElement.popoverTargetAction = 'toggle';
+
+    if (this.isOpen) {
+      this.popupElement.showPopover();
+      this.runUpdatePosition();
+    }
+  }
+
+  componentDidUpdate(): void {
+    if (this.isOpen) {
+      this.runUpdatePosition();
+    } else if (this.cleanupUpdatePosition) {
+      this.cleanupUpdatePosition();
+    }
+  }
+
+  disconnectedCallback(): void {
+    if (this.cleanupUpdatePosition) {
+      this.cleanupUpdatePosition();
+    }
+  }
+
+  private renderDismissButton(): JSX.Element {
+    if (this.displayDismissButton) {
+      return (
+        <gux-dismiss-button
+          onClick={this.dismiss.bind(this)}
+          position="inherit"
+        ></gux-dismiss-button>
+      ) as JSX.Element;
+    }
+  }
+
+  render(): JSX.Element {
+    return (
+      <popover
```

Comment:
Is this `<popover>` a valid tag ? I could not see it in the docs. All I could find was using the `popover` attribute like 
`<div id="mypopover" popover>Popover content</div>`

Reply:
Fixed